test/unit/db_redis_spec.js

```diff
@@ -150,14 +150,32 @@ describe('Testing REDIS database', function() {
         it('error getting token', function(done) {
             var implementations = {
                 smembers: function(key, callback) {
-                    return callback('Error');
+                    return callback('Error', null);
                 }
             }
             mocker(implementations, function(db, spies) {
-                db.getToken(function(err) {
+                db.getToken(function(err, token) {
                     assert.equal(err, 'Error');
+                    assert.equal(token, null);
                     assert.equal(spies.smembers.callCount, 1);
-                    assert.equal(spies.smembers.getCall(0).args[0], token);
+                    assert.equal(spies.smembers.getCall(0).args[0], 'token');
+                    done();
+                });
+            });
+        });
+
+        it('no available token', function(done) {
```

Comment:
Try to use more explicative names for the tests, something following the pattern "should ... when ..."